### DISCUSSION TITLE
essay/2024/12/10/top-down-approach

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#246] [Comments]
# essay/2024/12/10/top-down-approach

군대에 다녀와서 컴퓨터 공부를 처음 시작할 때 어떻게 해야 할지를 몰랐습니다.

https://jeho.page/essay/2024/12/10/top-down-approach.html

**Comment by kor-Chipmunk** [comment]
탑다운이 확실히 흥미를 많이 돋게하고, 그 추억으로 원동력이 많이 되는 것 같아요.
바톰업은 어렵지만 내공을 다지는 기초 공사라고 생각합니다.
그 어려운 수련을 버티셨으니 존경스럽습니다.
그때 그때 방향과 목적에 따라 탑다운과 바톰업을 적절히 사용할 줄 아는 사람이 되고 싶네요.
생소한 분야를 시작할 땐 탑다운으로, 어느정도 익힌다 싶으면 바톰업으로.

**Reply by BenjaminKim** [reply]
@kor-Chipmunk 그렇습니다. 적절히 섞어가며 공부해야 좋다고 생각합니다.
